calico/felix/fetcd.py

```diff
@@ -25,6 +25,7 @@
 import logging
 import socket
 import subprocess
+import pkg_resources  # part of setuptools
```

Comment:
There's more work to do to add a dependency than this.
- We support packaged installs for Ubuntu (14.04+16.04) and CentOS/RHEL(7): the .deb/.rpm files need to depend on python-setuptools (or whatever it's called).
- It needs to be added to the appropriate requirements.txt file.
- I think there's a frozen requirements.txt in libcalico (or somewhere) that this also needs to be added to.
- Worth grepping the main repos for another python dependency to see if I've missed anything.